components/dropdown/dropdown-content-mixin.js

```diff
@@ -228,13 +234,13 @@ export const DropdownContentMixin = superclass => class extends RtlMixin(supercl
 			maxWidth: this.maxWidth ? `${this.maxWidth}px` : undefined,
 			minWidth: this.minWidth ? `${this.minWidth}px` : undefined,
 			/* add 2 to content width since scrollWidth does not include border */
-			width: this._width ? `${this._width + 20}px` : undefined
+			width: this._width ? `${this._width + 20}px` : ''
 		};
 
 		const containerStyle = {
 			minWidth: this.minWidth ? `${this.minWidth}px` : undefined,
 			/* set width of content in addition to width container so IE will render scroll inside border */
-			width: this._width ? `${this._width + 18}px` : undefined,
+			width: this._width ? `${this._width + 18}px` : '',

```

Comment:
I'm not entirely sure why this change was necessary but with `undefined`, the width of the dropdown content would infinitely expand. This does not occur with `''`.

Reply:
Presumably with this change, `width=""` gets rendered in the inline styles, and it's essentially the same as `someElem.style.width = '';`?  What is the difference in computed value for the `width` in the browser?

Reply:
With `''` it looks like the width of the `d2l-dropdown-content-width` is always `370px` from the `width: 100vw` and `max-width: 370px` in the `dropdown-content-styles`. With `undefined` the `d2l-dropdown-content-width` drops down to the width of the `d2l-dropdown-content-container + 2` and triggers the `d2l-menu-resize` infinitely until it reaches the max of `370px`.

Interestingly, this only seems to show if I log with it `getComputedStyle().width`. If I try and observe it in the browser the computed style shows as `370px` from the `width: 100vw` for both `''` and `undefined`.